src/arch/riscv/insts/vector.hh

```diff
@@ -557,27 +573,29 @@ class VMaskMergeMicroInst : public VectorArithMicroInst
     Fault
     execute(ExecContext* xc, trace::InstRecord* traceData) const override
     {
-        vreg_t& tmp_d0 = *(vreg_t *)xc->getWritableRegOperand(this, 0);
+        vreg_t tmp_d0 = *(vreg_t *)xc->getWritableRegOperand(this, 0);
```

Comment:
@alvaromorenomv when merging this you undid a change that is required. This line should be:
`vreg_t& tmp_d0 = *(vreg_t *)xc->getWritableRegOperand(this, 0);`

Reply:
I think it is fixed now!